src/Selenium2Driver.php

```diff
@@ -107,45 +115,12 @@ public function setDesiredCapabilities($desiredCapabilities = null)
         }
 
         if (null === $desiredCapabilities) {
-            $desiredCapabilities = array();
+            $desiredCapabilities = [];
         }
 
         // Join $desiredCapabilities with defaultCapabilities
         $desiredCapabilities = array_replace(self::getDefaultCapabilities(), $desiredCapabilities);
 
-        if (isset($desiredCapabilities['firefox'])) {
```

Comment:
I'm not sure that we have tests to cover passing of browser-specific desired capabilities though. If we do, then they'll catch fact that something isn't passed.